src/metabase/api/api_key.clj

```diff
@@ -34,7 +34,8 @@
     (t2/with-transaction [_conn]
       (let [user (user/insert-new-user! {:email      email
                                          :first_name name
-                                         :type       :api-key})]
+                                         :type       :api-key
+                                         :is_active  false})]
```

Comment:
I wonder if this could have unintended side effects? It seems like API users should be considered active as long as the associated API key itself exists and can be used to issue requests. Maybe this doesn't break anything at the moment, but if someone builds a feature in the future and includes an `is-active` check for some reason, it would break for API users and they might not notice.

My bias would be to just adjust the `active-users-count` logic in `metabase.public-settings.premium-features` to explicitly exclude API keys. Wdyt?

I also realized that our anonymous stats reporting in `metabase.analytics.stats` includes more granular user count metrics. I wonder if we should update this to report the count of API keys separately. Probably a question for Arakaki.

Reply:
This makes sense to me, I changed `active-user-count`. For now, I just excluded API-key users from the stats in `metabase.analytics.stats` - if Arakaki wants us to we can add API Keys to the counts down the road, but I figured it's better to keep the status quo for now.